src/components/pages/AudioBook/rhap-style-override.css

```diff
@@ -0,0 +1,21 @@
+/* overiding border and background */
+.rhap_container {
+    background-color: transparent;
+    box-shadow: none;
+  }
+  
+  /* overiding progress bar colors and volume */
+  .rhap_progress-bar,
+  .rhap_download-progress,
+  .rhap_progress-filled,
+  .rhap_progress-bar-show-download {
+    background-color: dodgerblue;
+  }
+  
+  .rhap_volume-bar,
+  .rhap_progress-indicator,
+  .rhap_volume-indicator {
+    background: dodgerblue;
```

Comment:
is there a typical reason that you did just 'background' instead of 'background-color' like others?

Reply:
That's how react-h5-audio-player has it in their stylesheet.  I'm copying how they have it to override it.  This is there style sheet here for reference: https://github.com/lhz516/react-h5-audio-player/blob/master/src/styles.scss